frontend/src/metabase/embedding-sdk/config.ts

```diff
@@ -1,7 +1,5 @@
 export const EMBEDDING_SDK_ROOT_ELEMENT_ID = "metabase-sdk-root";
 export const EMBEDDING_SDK_PORTAL_ROOT_ELEMENT_ID = "metabase-sdk-portal-root";
-export const EMBEDDING_SDK_FULL_PAGE_PORTAL_ROOT_ELEMENT_ID =
```

Comment:
change from https://github.com/metabase/metabase/pull/60431, I'll remove it once that PR lands.